template/nuxt.config.js

```diff
@@ -1,4 +1,4 @@
-export default {
+module.exports = {
```

Comment:
```suggestion
exports default {
```

Why revert this line to ES5?

Reply:
we can use ES6 but still nuxt official uses ES5 that's why just sticking to the official one don't want to face more bugs you know